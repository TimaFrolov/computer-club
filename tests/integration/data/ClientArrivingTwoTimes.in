1
01:00 02:00
7
01:15 1 client1
01:30 1 client1
