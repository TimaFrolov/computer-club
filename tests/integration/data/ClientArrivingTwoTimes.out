01:00
01:15 1 client1
01:30 1 client1
01:30 13 YouShallNotPass
02:00 11 client1
02:00
1 0 00:00
